Backyarder/backyarder-ui
# Feature/connect sidebar api

## What I did:
UI is now serving the BE list data!
I also added `cache: 'force-cache'` option as a model on how to limit our external API calls if they have been called previously and are stored in cache.

Testing Notes:
- Ensure all Card icons and tooltips look correct
- Ensure all previous drag-and-drop functionality works

## Did this break anything?

## Type of change


## Checklist:

## What's next?
